src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -2225,8 +2226,8 @@ public void cannotCreateInstanceWithDuplicateHrid() throws Exception {
     final Response response = createCompleted.get(10, SECONDS);
 
     assertThat(response.getStatusCode(), is(400));
-    assertThat(response.getBody(),
-      is("HRID value already exists in table instance: in00000000001"));
+    assertTrue(response.getBody()
+      .contains("Key (lower(f_unaccent(jsonb ->> 'hrid'::text)))=(in00000000001) already exists."));
```

Comment:
I think that UI may expect the previous message format. So I would recommend to fix it.

Reply:
added separate method for handling hrid error cases for instances, can be replaced or modified with methods from PgUtil class.